change/@fluentui-react-spinner-08070866-8e5b-4438-a02f-3eeec8e27fb6.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29008&commitId=79f2862797496a8b849667f5fdbb30a2b53c360a&env=prod&clientType=FLUENTUI)


  <details><summary><b>Spinner converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Spinner converged.Primary - High Contrast.chromium.png|27| Changed|
|Spinner converged.Primary with Label - High Contrast.chromium.png|107| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>